x-pack/plugins/security/public/views/management/edit_role/components/privileges/kibana/privilege_callout_warning.js

```diff
@@ -0,0 +1,161 @@
+            <EuiCallOut color="warning" iconType="iInCircle" title={'Cannot customize a reserved role\'s space privileges'}>
+              <p>
+                This role always grants full access to all spaces.
+                To customize privileges for individual spaces, you must create a new role.
+              </p>
+            </EuiCallOut>
+            {this.getShowImpactedSpacesLink()}
+          </Fragment>
+        );
+      } else {
+        return (
+          <Fragment>
+            <EuiCallOut color="warning" iconType="iInCircle" title={'Minimum privilege is too high to customize individual spaces'}>
+              <p>
+                Setting the minimum privilege to <strong>all</strong> grants full access to all spaces.
+                To customize privileges for individual spaces,
+              the minimum privilege must be either <strong>read</strong> or <strong>none</strong>.
+              </p>
+            </EuiCallOut>
+            {this.getShowImpactedSpacesLink()}
+          </Fragment>
+        );
+      }
+    }
+
+    if (basePrivilege === 'read') {
+      if (isReservedRole) {
+        return (
+          <EuiCallOut color="warning" iconType="iInCircle" title={'Cannot customize a reserved role\'s space privileges'}>
+            <p>
+              This role always grants read access to all spaces.
+              To customize privileges for individual spaces, you must create a new role.
+            </p>
+          </EuiCallOut>
+        );
+      } else {
+        return (
+          <EuiCallOut color="primary" iconType="iInCircle" title={'Lowest possible privilege is \'read\''} />
+        );
+      }
+    }
+
+    if (basePrivilege === NO_PRIVILEGE_VALUE && isReservedRole) {
+      return (
+        <EuiCallOut color="warning" iconType="iInCircle" title={'Cannot customize a reserved role\'s space privileges'}>
+          <p>
+            This role never grants access to any spaces within Kibana.
+            To customize privileges for individual spaces, you must create a new role.
+          </p>
+        </EuiCallOut>
+      );
+    }
+
+    return null;
+  }
+
+  getFlyout = () => {
+    if (!this.state.showImpactedSpaces) {
+      return null;
+    }
+
+    const {
+      role,
+      spaces,
+    } = this.props;
+
+    const spacePrivileges = spaces.reduce((acc, space) => {
+      return {
+        ...acc,
+        [space.id]: ['all']
+      };
+    }, {});
+
+    return (
+      <EuiFlyout onClose={this.toggleShowImpactedSpaces} aria-labelledby="showImpactedSpacesTitle">
+        <EuiFlyoutHeader hasBorder>
+          <EuiTitle size="m">
+            <h1 id="showImpactedSpacesTitle">Summary of all space privileges</h1>
+          </EuiTitle>
+        </EuiFlyoutHeader>
+        <EuiFlyoutBody>
+          <PrivilegeSpaceTable readonly={true} role={role} spaces={spaces} spacePrivileges={spacePrivileges} />
+        </EuiFlyoutBody>
+        <EuiFlyoutFooter className="privilegeCallout--footer">
```

Comment:
I would just hide the entire footer if the manage button is not available.

Reply:
yep, once we have that mechanism in place, I'll go back and update this